src/client/utils/Market.ts

```diff
@@ -16,9 +16,9 @@ export default class MarketUtils {
                 value: listing.previousAvgPrice?.statValue ?? null,
                 date: DateUtils.convertToDate(listing.previousAvgPrice?.statDate ?? null),
             },
-            currentAvgHourPrice: {
-                value: listing.currentAvgHourPrice?.statValue ?? null,
-                date: DateUtils.convertToDate(listing.currentAvgHourPrice?.statDate ?? null),
+            currentHourPrice: {
```

Comment:
Please keep the old name and add a TODO for the next major release that we need to rename that variable to match the API fields